components/logonav/LogoNav.tsx

```diff
@@ -0,0 +1,17 @@
+import Image from 'next/image'
+import logo from '/public/assets/logo-colored-nav.svg'
+
+export const LogoNav = () => {
```

Comment:
@emestabillo whats the difference between this an the logo component you created?

Reply:
Em's logo component is for the main badge:
![Gridiron Survivor Badge Logo](https://github.com/LetsGetTechnical/gridiron-survivor/assets/40150036/8cb6f7c5-e2d2-4d77-bfc2-d96dfb0ae38e) one.
 
Mine is for the one that appears in the nav:
![Gridiron Survivor Text Logo](https://github.com/LetsGetTechnical/gridiron-survivor/assets/40150036/ed1f0440-bc29-4c94-8dd8-4ccd967465e1)